src/metabase/query_processor/util/add_alias_info.clj

```diff
@@ -67,6 +67,8 @@
   same unique alias."
   []
   (let [unique-name-fn (mbql.u/unique-name-generator
+                        ;; some databases treat aliases as case-insensitive so make sure the generated aliases are
+                        ;; unique regardless of case
```

Comment:
Adding extra comments about what's happening to code not actually touched as part of this PR